app/editor/menus/code.tsx

```diff
@@ -12,6 +14,26 @@ export default function codeMenuItems(
 ): MenuItem[] {
   const node = state.selection.$from.node();
 
+  const allLanguages = Object.entries(LANGUAGES);
+  const frequentLanguages = getFrequentCodeLanguages();
+
+  const frequentLangMenuItems = frequentLanguages.map((value) => {
```

Comment:
This is sorted by frequency.
Perhaps, alphabetical sort is better?

Reply:
Hard to say which is better to be honest, trying it out I think most recent works fine – I would start with this and we can change if there is any confusion